mergify_engine/worker.py

```diff
@@ -56,6 +57,43 @@ def event_handler(event_type, subscription, data):
     return real_event_handler(event_type, subscription, data)
 
 
+def private_installation_handler(installation_id):
+    """Create the initial configuration on all private repositories"""
+
+    integration = github.GithubIntegration(config.INTEGRATION_ID,
+                                           config.PRIVATE_KEY)
+    installation_token = integration.get_access_token(
+        installation_id).token
+    g = github.Github(installation_token)
+    try:
+        installation = g.get_installation(installation_id)
+        for repo in installation.get_repos():
+            if repo.private:
+                initial_configuration.create_pull_request_if_needed(
+                    installation_token, repo)
+    except github.RateLimitExceededException:
+        LOG.error("rate limit reached for install %d", installation_id)
+
+
+def installation_handler(installation, repository):
+    """Create the initial configuration on an repository"""
+
+    integration = github.GithubIntegration(config.INTEGRATION_ID,
+                                           config.PRIVATE_KEY)
+    installation_token = integration.get_access_token(
+        installation["id"]).token
+    g = github.Github(installation_token)
+    try:
+        user = g.get_user(installation["account"]["login"])
+        repo = user.get_repo(repository["name"])
+        initial_configuration.create_pull_request_if_needed(
+            installation_token, repo)
+    except github.RateLimitExceededException:
+        LOG.error("rate limit reached for install %d (%s)",
+                  installation["id"],
+                  repository["full_name"])
```

Comment:
there's a lot of duplicate code with the private version, and the error login seems different. not a big deal but just saying